cicd/pipelines/op-release-notes.adoc

```diff
@@ -41,14 +40,15 @@ The following compatibility matrix shows the support status and compatible versi
 |4.7
 
 |===
-////
 
 If you experience difficulty with a procedure described in this documentation,
 visit the Red Hat Customer Portal to learn more about link:https://access.redhat.com/support/offerings/techpreview/[Red Hat Technology Preview features support scope].
 
 For questions and feedback, you can send an email to the product team at pipelines-interest@redhat.com.
```

Comment:
@Preeticp - Should I remove this sentence too?

Reply:
No, let's retain that. We could check with the team if we want to add any other forum.

Reply:
is pipelines-interest@redhat.com a public mailing ?

Reply:
Not sure about that, @siamaksade can you throw some light on this, please? I found this: https://groups.google.com/a/redhat.com/g/pipelines-interest/about

Reply:
Is there some other forum like the google chat group that we want to highlight here?

Reply:
Actually it is, so we are good 😉